cmd/mqtt/main.go

```diff
@@ -122,6 +130,34 @@ func main() {
 		log.Fatalf(err.Error())
 	}
 
+	if cfg.mqttTargetHealthCheck != "" {
+		for i := 1; i <= maxHealthCheckRetries; i++ {
+			res, err := http.Get(cfg.mqttTargetHealthCheck)
+			if err != nil {
+				logger.Info(fmt.Sprintf("Broker not ready: %s ", err.Error()))
```

Comment:
Shouldn't we do `continue` here?

Reply:
I think not, without `continue` rest of the loop body will be skipped anyway, except `sleep` on the end. 
With `continue` next iteration would be right away.

Reply:
Yes, this is exactly what I meant - that the rest of the operations in this loop body should be skipped. Because if you can not do HTTP GET, do you really need to check the result? And do you really need to read HTTP body (remember that GET failed)? And then do you need to close the `Body` (remember, GET failed, and you did not read anything)?